### ISSUE TITLE
Razer Nommo Chroma Speaker support (1532:0517)

### ISSUE THREAD

**Issue by ghoshsomik** [openrazer/openrazer#714]
Razer Nommo Chroma
Product No: 
Model No: RZ05-02460100-R3U1

```bash
lsusb -d 1532:0517 -v                                             

Bus 001 Device 010: ID 1532:0517 Razer USA, Ltd 
Couldn't open device, some information will be missing
Device Descriptor:
  bLength                18
  bDescriptorType         1
  bcdUSB               2.00
  bDeviceClass            0 (Defined at Interface level)
  bDeviceSubClass         0 
  bDeviceProtocol         0 
  bMaxPacketSize0        64
  idVendor           0x1532 Razer USA, Ltd
  idProduct          0x0517 
  bcdDevice            2.00
  iManufacturer           1 
  iProduct                2 
  iSerial                 0 
  bNumConfigurations      1
  Configuration Descriptor:
    bLength                 9
    bDescriptorType         2
    wTotalLength          156
    bNumInterfaces          4
    bConfigurationValue     1
    iConfiguration          0 
    bmAttributes         0x80
      (Bus Powered)
    MaxPower              500mA
    Interface Descriptor:
      bLength                 9
      bDescriptorType         4
      bInterfaceNumber        0
      bAlternateSetting       0
      bNumEndpoints           1
      bInterfaceClass         3 Human Interface Device
      bInterfaceSubClass      0 No Subclass
      bInterfaceProtocol      1 Keyboard
      iInterface              0 
        HID Device Descriptor:
          bLength                 9
          bDescriptorType        33
          bcdHID               1.11
          bCountryCode            0 Not supported
          bNumDescriptors         1
          bDescriptorType        34 Report
          wDescriptorLength      58
         Report Descriptors: 
           ** UNAVAILABLE **
      Endpoint Descriptor:
        bLength                 7
        bDescriptorType         5
        bEndpointAddress     0x81  EP 1 IN
        bmAttributes            3
          Transfer Type            Interrupt
          Synch Type               None
          Usage Type               Data
        wMaxPacketSize     0x0010  1x 16 bytes
        bInterval               1
    Interface Descriptor:
      bLength                 9
      bDescriptorType         4
      bInterfaceNumber        1
      bAlternateSetting       0
      bNumEndpoints           0
      bInterfaceClass         3 Human Interface Device
      bInterfaceSubClass      0 No Subclass
      bInterfaceProtocol      0 None
      iInterface              0 
        HID Device Descriptor:
          bLength                 9
          bDescriptorType        33
          bcdHID               2.00
          bCountryCode            0 Not supported
          bNumDescriptors         1
          bDescriptorType        34 Report
          wDescriptorLength      22
         Report Descriptors: 
           ** UNAVAILABLE **
    Interface Descriptor:
      bLength                 9
      bDescriptorType         4
      bInterfaceNumber        2
      bAlternateSetting       0
      bNumEndpoints           0
      bInterfaceClass         1 Audio
      bInterfaceSubClass      1 Control Device
      bInterfaceProtocol      0 
      iInterface              0 
      AudioControl Interface Descriptor:
        bLength                 9
        bDescriptorType        36
        bDescriptorSubtype      1 (HEADER)
        bcdADC               1.00
        wTotalLength           43
        bInCollection           1
        baInterfaceNr( 0)       3
      AudioControl Interface Descriptor:
        bLength                12
        bDescriptorType        36
        bDescriptorSubtype      2 (INPUT_TERMINAL)
        bTerminalID             1
        wTerminalType      0x0101 USB Streaming
        bAssocTerminal          0
        bNrChannels             2
        wChannelConfig     0x0003
          Left Front (L)
          Right Front (R)
        iChannelNames           0 
        iTerminal               0 
      AudioControl Interface Descriptor:
        bLength                13
        bDescriptorType        36
        bDescriptorSubtype      6 (FEATURE_UNIT)
        bUnitID                 2
        bSourceID               1
        bControlSize            2
        bmaControls( 0)      0x01
        bmaControls( 0)      0x00
          Mute Control
        bmaControls( 1)      0x02
        bmaControls( 1)      0x00
          Volume Control
        bmaControls( 2)      0x02
        bmaControls( 2)      0x00
          Volume Control
        iFeature                0 
      AudioControl Interface Descriptor:
        bLength                 9
        bDescriptorType        36
        bDescriptorSubtype      3 (OUTPUT_TERMINAL)
        bTerminalID             3
        wTerminalType      0x0301 Speaker
        bAssocTerminal          0
        bSourceID               2
        iTerminal               0 
    Interface Descriptor:
      bLength                 9
      bDescriptorType         4
      bInterfaceNumber        3
      bAlternateSetting       0
      bNumEndpoints           0
      bInterfaceClass         1 Audio
      bInterfaceSubClass      2 Streaming
      bInterfaceProtocol      0 
      iInterface              0 
    Interface Descriptor:
      bLength                 9
      bDescriptorType         4
      bInterfaceNumber        3
      bAlternateSetting       1
      bNumEndpoints           1
      bInterfaceClass         1 Audio
      bInterfaceSubClass      2 Streaming
      bInterfaceProtocol      0 
      iInterface              0 
      AudioStreaming Interface Descriptor:
        bLength                 7
        bDescriptorType        36
        bDescriptorSubtype      1 (AS_GENERAL)
        bTerminalLink           1
        bDelay                  0 frames
        wFormatTag              1 PCM
      AudioStreaming Interface Descriptor:
        bLength                11
        bDescriptorType        36
        bDescriptorSubtype      2 (FORMAT_TYPE)
        bFormatType             1 (FORMAT_TYPE_I)
        bNrChannels             2
        bSubframeSize           2
        bBitResolution         16
        bSamFreqType            1 Discrete
        tSamFreq[ 0]        48000
      Endpoint Descriptor:
        bLength                 9
        bDescriptorType         5
        bEndpointAddress     0x03  EP 3 OUT
        bmAttributes            9
          Transfer Type            Isochronous
          Synch Type               Adaptive
          Usage Type               Data
        wMaxPacketSize     0x00c4  1x 196 bytes
        bInterval               1
        bRefresh                0
        bSynchAddress           0
        AudioControl Endpoint Descriptor:
          bLength                 7
          bDescriptorType        37
          bDescriptorSubtype      1 (EP_GENERAL)
          bmAttributes         0x01
            Sampling Frequency
          bLockDelayUnits         0 Undefined
          wLockDelay              0 Undefined
 ```
```bash
 $ ls -lah /dev/input/by-id/                                         [14:10:08]

total 0
drwxr-xr-x 2 root root 400 Nov 17 14:04 .
drwxr-xr-x 4 root root 780 Nov 17 14:04 ..
lrwxrwxrwx 1 root root   9 Nov 17 10:43 usb-11121119-000JJ741X_USB_Camera_200901010001-event-if00 -> ../event7
lrwxrwxrwx 1 root root  10 Nov 17 13:52 usb-C-Media_Electronics_Inc._USB_Advanced_Audio_Device-event-if03 -> ../event13
lrwxrwxrwx 1 root root  10 Nov 17 10:43 usb-Razer_Razer_BlackWidow_Ultimate_2016-event-kbd -> ../event25
lrwxrwxrwx 1 root root  10 Nov 17 10:43 usb-Razer_Razer_BlackWidow_Ultimate_2016-if01-event-kbd -> ../event19
lrwxrwxrwx 1 root root  10 Nov 17 10:43 usb-Razer_Razer_BlackWidow_Ultimate_2016-if02-event-mouse -> ../event20
lrwxrwxrwx 1 root root   9 Nov 17 10:43 usb-Razer_Razer_BlackWidow_Ultimate_2016-if02-mouse -> ../mouse3
lrwxrwxrwx 1 root root  10 Nov 17 10:43 usb-Razer_Razer_Blade-event-kbd -> ../event22
lrwxrwxrwx 1 root root  10 Nov 17 10:43 usb-Razer_Razer_Blade-if01-event-kbd -> ../event23
lrwxrwxrwx 1 root root   9 Nov 17 10:43 usb-Razer_Razer_Blade-if02-event-mouse -> ../event8
lrwxrwxrwx 1 root root   9 Nov 17 10:43 usb-Razer_Razer_Blade-if02-mouse -> ../mouse0
lrwxrwxrwx 1 root root  10 Nov 17 13:52 usb-Razer_Razer_Firefly-event-mouse -> ../event28
lrwxrwxrwx 1 root root  10 Nov 17 13:52 usb-Razer_Razer_Firefly-if01-event-kbd -> ../event26
lrwxrwxrwx 1 root root  10 Nov 17 13:52 usb-Razer_Razer_Firefly-if02-event-kbd -> ../event27
lrwxrwxrwx 1 root root   9 Nov 17 13:52 usb-Razer_Razer_Firefly-mouse -> ../mouse4
lrwxrwxrwx 1 root root  10 Nov 17 13:57 usb-Razer_Razer_Mamba-event-mouse -> ../event21
lrwxrwxrwx 1 root root  10 Nov 17 13:57 usb-Razer_Razer_Mamba-if01-event-kbd -> ../event17
lrwxrwxrwx 1 root root   9 Nov 17 13:57 usb-Razer_Razer_Mamba-mouse -> ../mouse1
lrwxrwxrwx 1 root root  10 Nov 17 14:04 usb-Razer_Razer_Nommo_Chroma-event-if00 -> ../event10
```

Synapse Beta ScreenShots:
![image](https://user-images.githubusercontent.com/5521495/48666438-fc9d2e00-ea76-11e8-9234-1d2ca617fe2b.png)
![image](https://user-images.githubusercontent.com/5521495/48666447-1d658380-ea77-11e8-8744-9f3b36a75cfb.png)

**Comment by z3ntu** [comment]
Please read [this](https://github.com/openrazer/openrazer/wiki/Reverse-Engineering-USB-Protocol) because we need packet captures.

**Comment by z3ntu** [comment]
Fixed with #1061
